components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -457,13 +465,25 @@ private static CACertificate resourceToCACertObject(Resource resource) throws Ce
     }
 
     private static void addDefaultCACertificateInRegistry(Registry registry, String caCertRegPath,
-                                                          X509Certificate certificate)
+                                                          X509Certificate certificate, OMElement validatorChildElement)
             throws CertificateValidationException {
 
+        List<String> ocspUrls = new ArrayList<>();
+        List<String> crlUrls = new ArrayList<>();
         try {
             if (!registry.resourceExists(caCertRegPath)) {
+                List<Validator> defaultValidatorConfig = getDefaultValidatorConfig(validatorChildElement);
+                for (Validator validator : defaultValidatorConfig) {
+                    if (validator.isEnabled()) {
+                        if (X509CertificateValidationConstants.OCSP_VALIDATOR.equals(validator.getDisplayName())) {
+                            ocspUrls = getAIALocations(certificate);
+                        }
+                        if (X509CertificateValidationConstants.CRL_VALIDATOR.equals(validator.getDisplayName())) {
```

Comment:
Since we check the value of 'validator.getDisplayName()' in both these if statements, can't we use 'else if' for the latter if check?

Reply:
Fixed https://github.com/wso2-extensions/identity-x509-commons/pull/20/commits/aa080123a718d304178d0ec2b6e299a54d7eef29